stakerservice/service.go

```diff
@@ -125,6 +125,70 @@ func (s *StakerService) stake(_ *rpctypes.Context,
 	}, nil
 }
 
+func (s *StakerService) btcDelegationFromBtcStakingTx(
+	_ *rpctypes.Context,
```

Comment:
nit: rm param if not needed

Reply:
this param is needed even if not used due to the interface of rpc